src/com/xilinx/rapidwright/design/blocks/PBlockGenerator.java

```diff
@@ -157,6 +158,10 @@ private void getResourceUsages(String reportFileName){
 			}
 		}
 		
+		if (dev.getName().contains("xc7")) {
```

Comment:
Some Series 7 devices don't actually match.  Try this instead:

`if (dev.getSeries() == Series.Series7) {`